config/github-labels.yml

```diff
@@ -0,0 +1,152 @@
+# List of labels to remove: we’re not using anything like this anymore.
+remove:
```

Comment:
is `remove` intended to be a permanent part of the config?

Reply:
`remove` can be removed, probably, when all repos are updated.

`legacy` includes GitHub defaults, so we keep them: When a new repo is added to the orgs in the future, their labels are used but changed (new name, color, description)